### PR TITLE
Staffbot usability updates.

### PR DESCRIPTION
**Pull request by kkamalov**
screen shot 2016-05-18 at 2 33 43 pm

### CODE REVIEW

File: orchestra/bots/staffbot.py [hunk 1]
```diff
@@ -110,17 +124,28 @@ def restaff(self, task_id, username,
         The current user for the given username is removed, and a new user
         is found.
         """
-        # TODO(kkamalov): maybe username could also be slack_username
+        command = 'restaff {} {}'.format(task_id, username)
         try:
-            worker = Worker.objects.get(user__username=username)
```
Comment (joshblum) [inline orchestra/bots/staffbot.py:115]: use `Q` operators and just do a single or query so we don't hit the DB twice :)

File: orchestra/bots/tests/test_basebot.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def test_dispatch(self):
             (r'test_command (?P<test_param>[0-9]+)', 'test_command'),
         )
 
-        def test_command(test_param):
+        def test_command(test_param, command=None):
```
Comment (joshblum) [inline orchestra/bots/tests/test_basebot.py]: what is this for?

File: orchestra/bots/staffbot.py [hunk 3]
```diff
@@ -101,7 +109,13 @@ def staff(self, task_id,
 
         slack_message = self.staffing_success.format(task_id)
         message_experts_slack_group(task.project.slack_group_id, slack_message)
-        return format_slack_message(slack_message)
+        return format_slack_message(
+            command,
+            attachments=[{
+                'color': 'good',
```
Comment (joshblum) [inline orchestra/bots/staffbot.py:116]: can we abstract this to a success/error attachement message so we don't have to repeat? or maybe it's not worth it?

**Comment by joshblum** [comment]
love the new slack messaging look :)

File: orchestra/bots/staffbot.py (hunk 3 above)
Comment (kkamalov) [inline orchestra/bots/staffbot.py:116]: i dont think it's worth it :P

File: orchestra/bots/tests/test_basebot.py [hunk 4]
```diff
@@ -44,7 +44,7 @@ def test_dispatch(self):
...
             return text
```
Comment (joshblum) [inline orchestra/bots/tests/test_basebot.py]: also how does this function work haha what is `text` should that be `test_param`?

Comment (kkamalov) [inline orchestra/bots/tests/test_basebot.py]: lol. no idea? did you write the code ;P?

Comment (kkamalov) [inline orchestra/bots/tests/test_basebot.py]: text is actually defined couple of lines below

File: orchestra/bots/tests/test_basebot.py [hunk 5]
```diff
@@ -44,7 +44,7 @@ def test_dispatch(self):
             (r'test_command (?P<test_param>[0-9]+)', 'test_command'),
         )
 
-        def test_command(test_param, command=None):
+        def test_command(test_param):
             return text
```
Comment (joshblum) [inline orchestra/bots/tests/test_basebot.py]: should this return 'test_param'?

**Comment by joshblum** [comment]
lgtm after tests

**Pull request merged**